### ISSUE TITLE
Accessibility - Missing form label on Thanks you page

### ISSUE THREAD

**Issue by lmcbhvu** [cds-snc/report-a-cybercrime#2393]
## Summary

Got an error (Missing form label) when runs WAVE on Thank you page

![image](https://user-images.githubusercontent.com/60269104/96631796-cc0a4280-12e4-11eb-86f5-6e26966514fd.png)

## Steps to reproduce

> How exactly can the issue be reproduced? Be very specific. Include any plugins used to find the bug.

## Unresolved questions

> Are there any related issues you consider out of scope for this issue that could be addressed in the future?

**Comment by DianeLiu2019** [comment]
@lmcbhvu  Please check if this issue still exist. I tested the error message is gone. It should be fixed in PR# 2373.

**Comment by lmcbhvu** [comment]
I re-test it and the error is gone. This defect is closed
